spark/src/main/scala/org/apache/spark/sql/delta/DeltaAnalysis.scala

```diff
@@ -147,6 +147,8 @@ class DeltaAnalysis(session: SparkSession)
           // maxColumnId will be set in the targetTable's configuration internally after
           val config =
             deltaLogSrc.snapshot.metadata.configuration.-("delta.columnMapping.maxColumnId")
+              .-(MaterializedRowId.MATERIALIZED_COLUMN_NAME_PROP)
```

Comment:
we should also reflect in the comment that more fields cannot be set now

Reply:
I updated the comment